technic/machines/LV/geothermal.lua

```diff
@@ -26,8 +27,10 @@ minetest.register_craftitem("technic:geothermal", {
 
 local check_node_around = function(pos)
 	local node = minetest.get_node(pos)
-	if node.name == "default:water_source" or node.name == "default:water_flowing" then return 1 end
-	if node.name == "default:lava_source"  or node.name == "default:lava_flowing"  then return 2 end
+	if node.name == "default:water_source" or node.name == "default:water_flowing"
+		or node.name == "mcl_core:water_source" or node.name == "mcl_core:water_flowing" then return 1 end
+	if node.name == "default:lava_source"  or node.name == "default:lava_flowing"
+		or node.name == "mcl_core:lava_source" or node.name == "mcl_core:lava_flowing" then return 2 end
```

Comment:
Maybe define these like it was done elsewhere, something like:
```lua
local water_source = has_mcl and "mcl_core:water_source" or "default:water_source"
```
I guess water sources could also go into materials.lua and use `mat.water_source` / `mat.water_flowing`... same for lava.

Reply:
done